src/main/java/com/ldtteam/structurize/blocks/decorative/BlockTimberFrame.java

```diff
@@ -1,26 +1,21 @@
 package com.ldtteam.structurize.blocks.decorative;
 
-import com.ldtteam.structurize.blocks.AbstractBlockStructurize;
 import com.ldtteam.structurize.blocks.types.TimberFrameCentreType;
-import com.ldtteam.structurize.blocks.types.TimberFrameFrameType;
 import com.ldtteam.structurize.blocks.types.TimberFrameType;
+import com.ldtteam.structurize.blocks.types.WoodType;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
+import net.minecraft.block.DirectionalBlock;
 import net.minecraft.block.material.Material;
 import net.minecraft.item.BlockItemUseContext;
-import net.minecraft.state.DirectionProperty;
 import net.minecraft.state.StateContainer;
-import net.minecraft.state.properties.BlockStateProperties;
 import net.minecraft.util.Mirror;
-import net.minecraft.util.Rotation;
 
 /**
  * Decorative block
  */
-public class BlockTimberFrame extends AbstractBlockStructurize<BlockTimberFrame>
+public class BlockTimberFrame extends DirectionalBlock
```

Comment:
Is this safe in terms of block data?

Reply:
The only difference is that the directional property now exists in the vanilla superclass - so that should not affect block data at all.